src/Microsoft.ML.LightGbm/LightGbmBinaryTrainer.cs

```diff
@@ -162,7 +162,7 @@ public enum EvaluateMetricType
             [Argument(ArgumentType.AtMostOnce,
                 HelpText = "Evaluation metrics.",
                 ShortName = "em")]
-            public EvaluateMetricType EvaluationMetric = EvaluateMetricType.Logloss;
+            public EvaluateMetricType EvaluationMetric = EvaluateMetricType.Default;
```

Comment:
>Default [](start = 76, length = 7)

Isn't this a breaking change?
cc @eerhardt

Reply:
Yes, but the other option is to have an inconsistent user experience. I talked to @ebarsoumMS about this. Let's discuss and reach a conclusion.

Reply:
It's not an "API breaking change". I think it falls into the scenarios that @TomFinley listed here https://github.com/dotnet/machinelearning/issues/3602#issuecomment-487192476.

> However even many years later sometimes we still have somewhat troublesome defaults running around

Here, if there is a better default value, I think it is acceptable to change the default.

Reply:
Also, this doesn't actually change training behavior, nor the metrics calculated by ML.NET evaluators. Just changes the metric that LightGbm calculates internally. 

In ML.NET, when we do the following (e.g. for binary classification)
```csharp
var transformedTestData = model.Transform(testData);
var metrics = mlContext.BinaryClassification.Evaluate(transformedTestData);
```
the evaluator computes all relevant metrics for binary classification regardless of what is specified by LightGbm's `EvaluationMetric` parameter.

Reply:
It may control LightGBM's early stopping, but otherwise I think this is a NOOP change. ML.NET doesn't relay the stdout from LightGBM to the user, and ML.NET uses its own evaluators for computing the final metrics.

Users could benefit from ML.NET relaying this info back to the user. This would allow a GUI to show the learning curves in real time (or as text output from a CLI):
[<img src="https://github.com/catboost/catboost/raw/master/catboost/benchmarks/gpu_vs_cpu_training_speed/fig/auc_vs_iteration.png">](https://github.com/catboost/catboost/tree/master/catboost/benchmarks/gpu_vs_cpu_training_speed)